src/traces/treemap/attributes.js

```diff
@@ -141,7 +141,19 @@ module.exports = {
 
         line: sunburstAttrs.marker.line,
 
-        editType: 'calc'
+        fillet: {
+            valType: 'number',
+            min: 0,
+            // max: 10, // TODO: Do we need a max?
```

Comment:
Looks like it's possible to make the radius so big that the text or smaller inner boxes poke out of their containers. Here's your mock bumped up to radius 40:
<img width="735" alt="Screen Shot 2022-10-27 at 19 19 21" src="https://user-images.githubusercontent.com/2678795/198415562-2fc5ce9f-652b-4963-9389-34b126530380.png">
How about we make a dynamic `max`? If I did the math right, a safe value would be `x+y+sqrt(2*x*y)` where `x` is `max(padding.l, padding.r)` and `y` is `max(padding.t, padding.b)`
Actually might be more user-friendly to leave `max` undefined, but restrict it during rendering and describe this in the attribute description - since we're restricting the radius anyway if the box gets too small... something like "the actual radius we draw will be limited to half the box size, or smaller if necessary to prevent the box contents from overflowing its edges"